### PR TITLE
Fixes for #197 and #198, Addresses issues of displaying CorpusSentences

### PR DESCRIPTION
**Pull request by ablodge**
This code adds a field `deprecated` to Corpus models, makes Corpus models' metadata editable, adds a filter to sentence display tables to only include non-deprecated corpora in the display, and fixes the CorpusSentence view to always display the sentence from the correct corpus version.

### CODE REVIEW

File: src/wiki/plugins/metadata/migrations/0047_auto_20200808_2256.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.13 on 2020-08-09 02:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (nschneid) [inline src/wiki/plugins/metadata/migrations/0047_auto_20200808_2256.py:6]: There are 2 new migrations for 1 new field. Could you please delete both these files and rerun makemigrations?

File: src/wiki/plugins/metadata/tables.py [hunk 2]
```diff
@@ -129,29 +129,28 @@ def tokens_for_supersense(article_id):
     t = PTokenAnnotation.objects.select_related('construal__article__current_revision',
                                                 'construal__role__article__current_revision', 'construal__function__article__current_revision',
                                                 'usage__current_revision__metadatarevision__usagerevision__article_revision__article',
-                                                'sentence', 'adposition').filter(Q(construal__role__article__id=article_id) | Q(construal__function__article__id=article_id)).order_by('sentence__sent_id')
+                                                'sentence', 'adposition').filter(Q(sentence__corpus__deprecated=False), Q(construal__role__article__id=article_id) | Q(construal__function__article__id=article_id)).order_by('sentence__sent_id')
```
Comment (nschneid) [inline src/wiki/plugins/metadata/tables.py:132]: Is the `Q()` wrapper necessary for the first item? My guess is it's only necessary for the disjunction.

File: src/wiki/plugins/metadata/models.py [hunk 3]
```diff
@@ -858,11 +858,16 @@ class Corpus(SimpleMetadata):
...
     languages = models.CharField(max_length=200, null=True, verbose_name="Language(s)")
-
+    deprecated = models.BooleanField(default=False, verbose_name="Is this a deprecated version of a corpus?")
 
     def __str__(self):
         return self.name.lower() + self.version
 
+    @classmethod
+    def editurl(cls, urlpath):
+        # return "_plugin/metadata/editp"
```
Comment (nschneid) [inline src/wiki/plugins/metadata/models.py:868]: editp is for adpositions, right?

**Comment by nschneid** [comment]
In the corpus import instructions, please add a note that if this is a newer version of an existing corpus, the old one should be marked as deprecated so its annotations will be hidden by default.

File: src/wiki/plugins/metadata/models.py (hunk 3 above)
Comment (ablodge) [reply src/wiki/plugins/metadata/models.py:868]: Yes, that's a copy/paste error.

File: src/wiki/plugins/metadata/tables.py (hunk 2 above)
Comment (ablodge) [reply src/wiki/plugins/metadata/tables.py:132]: Yes, it is necessary. Notice that usually `filter` takes named arguments, but `Q` objects are taken as positional arguments which in python must come first. See [complex-lookups-with-q-objects](https://docs.djangoproject.com/en/3.0/topics/db/queries/#complex-lookups-with-q-objects). You might be able to mix argument types, but I think this is cleaner.

File: src/wiki/plugins/metadata/migrations/0047_auto_20200808_2256.py (hunk 1 above)
Comment (ablodge) [reply src/wiki/plugins/metadata/migrations/0047_auto_20200808_2256.py]: Sure

**Comment by ablodge** [comment]
> For completeness the corpus article page should display a deprecation banner if the corpus is deprecated, saying something like "There is a newer version of this corpus. Annotations from this version will not appear on documentation pages by default."

Alright. For now, I added the message to the function `corpus_stats`. It displays at the top if the corpus is deprecated. Does that sound fine?

**Pull request merged**